javascript/pull-view.js

```diff
@@ -29,13 +29,14 @@
         this.$el.addClass('unimportant-repo');
       }
 
-      if (this.model.comment.get('thumbsup')) {
-        this.$el.addClass("thumbsup");
+      if (this.model.comment.get('thumbsup') || this.model.reviewComment.get('approved')) {
```

Comment:
It's possible that a PR has been approved by one person, but changes requested by another person. In this case both classes would be applied I think which wouldn't be good?